tpm/src/lib.rs

```diff
@@ -7,4 +7,5 @@
 extern crate log;
 
 pub mod socket;
+pub mod tpm_emulator;
```

Comment:
Do you really need these prefixes? We are already in the tpm crate.

Reply:
dropped the `tpm_` prefix